notebooks/community/managed_notebooks/pricing_optimization/pricing-optimization.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<p>Data is in inaccessible bucket. Move to gs://cloud-samples-data/ai-platform-unified/datasets/tabular</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/249/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
You can open access for me and I'll copy it over.